6.js

```diff
@@ -0,0 +1,31 @@
+function pathTo4chan (jsonObject) {
+  const object = JSON.parse(jsonObject)
+
+  const look = lookFor4chan(object)
+
+  if (look.found) {
+    return look.path.reverse().join(' -> ')
+  } else {
+    return '4chan not found'
+  }
+}
+
+function lookFor4chan (input) {
+  if (input === '4chan') {
+    return {found: true, path: []}
+  } else if (typeof input !== 'object') {
+    return {found: false, path: []}
+  } else {
```

Comment:
if you change this to `else if (typeof input === 'object')`, then you can get get rid of the other else if on line 16. It will hit your return on line 27.